tests/Domain/Insights/SyntaxCheckTest.php

```diff
@@ -33,23 +34,49 @@ public function testHasIssueOnFile(): void
 
     public function testHasIssueOnDirectory(): void
     {
+        $basepath = __DIR__ . '/Fixtures/InvalidPhpCode';
         /** @var \NunoMaduro\PhpInsights\Domain\Insights\InsightCollection $insights */
         $insights = $this->runAnalyserOnPreset(
             'default',
-            [__DIR__ . '/Fixtures/InvalidPhpCode/SemiColonAfterExtendClass.php'],
+            [$basepath . '/SemiColonAfterExtendClass.php', $basepath . '/UnclosedComment.php'],
```

Comment:
Changes in #448 mean when there's only one file in the collector, the command will *always* be generated using that file. Without a second file here, the test is essentially identical to the file one.